ShelbyFinancials/ReadMe.md

```diff
@@ -0,0 +1,145 @@
+  - **New Block**:  Shelby Financials Batches to Journal (added to the Shelby GL Batch Export page on install) 
+  - **New Account Attributes**: There are a number of new Account attributes that control where transactions are posted in Shelby Financials
+  - **New Transaction Attribute**: Transaction Project
+  - **New Page**: Projects (Finance > Administration > Projects)
+  - **New Page**: Shelby GL Export (Finance > Functions > Shelby GL Export)  
+  - **New Defined Type**: Financial Projects stores the Defined Values that designate what Project a transaction or batch should be associated with  
+  - **New Batch Attribute**: Date Exported  
+
+
+
+## Configuration
+
+#### Batch to Journal Block
+
+After install, the Shelby Financials Batch to Journal block was added to your Batch Details page. 
+
+![](https://user-images.githubusercontent.com/81330042/113193210-53a99500-9225-11eb-9066-85ebd714e6ae.png)
+
+
+
+
+
+
+
+
+![](Images/BatchToJournalProperties.png)
+
+> <span style="padding-left: 30px; margin-right: 10px; width: .8em;background: #d21919; border-radius: 100%; color: white; text-align: center; display: inline-block;">&nbsp;&nbsp;1</span>**Name** Block name
+>
+> <span style="padding-left: 30px; margin-right: 10px; width: .8em;background: #d21919; border-radius: 100%; color: white; text-align: center; display: inline-block;">&nbsp;&nbsp;2</span>**Button Text** Customize the text on the export button
+>
+> <span style="padding-left: 30px; margin-right: 10px; width: .8em;background: #d21919; border-radius: 100%; color: white; text-align: center; display: inline-block;">&nbsp;&nbsp;3</span>**Close Batch** Control whether the batch gets marked as closed in Rock after export
+
+
+
+#### Shelby Financials Batches to Journal Block
+
+![](https://user-images.githubusercontent.com/81330042/113193298-6fad3680-9225-11eb-95ef-f61edf9b87a7.png)
+
+
+> <span style="padding-left: 30px; margin-right: 10px; width: .8em;background: #d21919; border-radius: 100%; color: white; text-align: center; display: inline-block;">&nbsp;&nbsp;1</span>**Name** Block name
+>
+> <span style="padding-left: 30px; margin-right: 10px; width: .8em;background: #d21919; border-radius: 100%; color: white; text-align: center; display: inline-block;">&nbsp;&nbsp;2</span>**Detail Page** Link to the Financial Batch Details page
+>
+> <span style="padding-left: 30px; margin-right: 10px; width: .8em;background: #d21919; border-radius: 100%; color: white; text-align: center; display: inline-block;">&nbsp;&nbsp;3</span>**Button Text** Customize the text for the export button
+>
+> <span style="padding-left: 30px; margin-right: 10px; width: .8em;background: #d21919; border-radius: 100%; color: white; text-align: center; display: inline-block;">&nbsp;&nbsp;4</span>**Months Back** Number of months back that batches should be loaded. This is helpful to prevent database timeouts if there are years of historical batches
```

Comment:
```suggestion
> <span style="padding-left: 30px; margin-right: 10px; width: .8em;background: #d21919; border-radius: 100%; color: white; text-align: center; display: inline-block;">&nbsp;&nbsp;4&nbsp;&nbsp;</span>**Months Back** Number of months back that batches should be loaded. This is helpful to prevent database timeouts if there are years of historical batches
```